inst/qml/mlPrediction.qml

```diff
@@ -50,7 +50,7 @@ Form
 		{
 			id:									predictors
 			name:								"predictors"
```

Comment:
One of our guidelines will be that the variableName should closely resemble that of its title.